RahiDroid/gutenberg
# Post Featured Image block: Enable gradient overlay

## What?

- Adds the `Overlay` settings (color & opacity) to the Post featured block.

## Why?
- There's a need for the Post featured block to have the overlay settings in TT3 and generally, too, it's good to have. 
- Checkout: https://github.com/WordPress/gutenberg/issues/43676


## How?


## Testing Instructions




1. Open the post editor or the site editor.
1. Insert the `Post Featured Block`.
1. You should see the `Overlay` controls in the inspector controls of the block.
1. Applying the overlay should reflect, both in the editor and on the front-end.

## Screenshots or screencast 
![Sep-03-2022 00-15-46](https://user-images.githubusercontent.com/59014930/188218200-38c5465f-c0a8-40eb-b9ac-2191a2bc9e6d.gif)
